pages/packages.md

```diff
@@ -49,14 +49,14 @@ Some advanced package creation tools include:
 
 Currently, Buildkite Packages supports the following package ecosystems:
 
-- Alpine
-- Docker container images
-- Debian (deb for Debian and Ubuntu)
-- Ruby (RubyGems gem)
+- Alpine (apk)
+- Container (Docker) images
+- Debian (and Ubuntu)
```

Comment:
Hmm, most of these have the package type in the parentheses. Maybe something like `Debian and Ubuntu (DEB)` could be more consistent?